home/tests.py

```diff
@@ -1,3 +1,35 @@
-# from django.test import TestCase
+from home.models import Profile, Question, Answer
+import pytest
 
-# Create your tests here.
+
+@pytest.fixture
+def profile():
+    profile = Profile.create(username="test_user", password="testtest", email="test@test.com")
+    profile.save()
```

Comment:
You did profile.save() inside the Profile.create already

Reply:
Thanks, removed it